paper/paper.md

```diff
@@ -0,0 +1,174 @@
+    orcid: 0000-0001-7259-6120
+  - affiliation: 9
+    name: Kambiz Tavabi
+    orcid: 0000-0003-1881-892X
+  - affiliation: 10
+    name: Richard Höchenberger
+    orcid: 0000-0002-0380-4798
+  - affiliation: 11
+    name: Dominik Welke
+    orcid: 0000-0002-5529-1998
+  - affiliation: 12
+    name: Clemens Brunner
+    orcid: 0000-0002-6030-2233
+  - affiliation: 13
+    name: Alexander P Rockhill
+    orcid: 0000-0003-3868-7453
+  - affiliation: 9
+    name: Eric Larson
+    orcid: 0000-0003-4782-5360
+  - affiliation: 14
+    name: Alexandre Gramfort
+    orcid: 0000-0001-9791-4404
+  - affiliation: 15
+    name: Mainak Jas
+    orcid: 0000-0002-3199-9027
+affiliations:
+- index: 1
+  name: Center for Adaptive Rationality, Max Planck Institute for Human Development, Berlin, Germany
+- index: 2
+  name: Department of Cognitive Sciences, Macquarie University, Sydney, Australia
+- index: 3
+  name: Mozilla
+- index: 4
+  name: Department of Neuroscience and Biomedical Engineering, Aalto University, Espoo, Finland
+- index: 5
+  name: Human Cortical Physiology and Neurorehabilitation Section, NINDS, NIH, Bethesda, Maryland 20892
+- index: 6
+  name: UC Berkeley, Project Jupyter
+- index: 7
+  name: Institut du cerveau et de la moelle épinière (ICM), Paris, France
+- index: 8
+  name: Department of Biomedical and Clinical Sciences 'L. Sacco', University of Milan, Milan, Italy
+- index: 9
+  name: Institute for Learning and Brain Sciences, University of Washington, Seattle, WA, USA
+- index: 10
+  name: Institute of Neuroscience and Medicine (INM-3), Research Center Jülich, Germany
+- index: 11
+  name: Max-Planck-Institute for Empirical Aesthetics, Frankfurt a.M., Germany
+- index: 12
+  name: Institute of Psychology, University of Graz, Austria
+- index: 13
+  name: University of Oregon, Eugene OR, USA
+- index: 14
+  name: Université Paris-Saclay, Inria, CEA, Palaiseau, France
+- index: 15
+  name: Athinoula A. Martinos Center for Biomedical Imaging, Massachusetts General Hospital, Charlestown, MA, USA
+date: 29 October 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+With the development of the Brain Imaging Data Structure
+(``BIDS``; [@Gorgolewski2016]), the neuroscientific community received
+a standard to organize and share data. BIDS standardizes storage of complex experimental
+data by prescribing a set of file naming conventions and templates to store the
+associated metadata in the form of Javascript Object Notation (JSON) files. It thus
+facilitates data sharing, eases metadata querying and enables automatic data
+analysis pipelines.
+
+While BIDS was originally intended for Magnetic Resonance Imaging (MRI)
+data, it has recently been extended for electro-/magneto-encephalography (EEG/MEG)
+[@Niso2018, @Pernet2019], and intracranial EEG (iEEG) [@Holdgraf2019].
+In this context, many software packages for electrophysiology data analysis are
+now becoming BIDS compatible. BIDS is a rich system to curate, aggregate and
```

Comment:
In the current phrasing, this sentence jumps out of nowhere. I wanted to make the point that BIDS really allows you to do things more than what `.fif`, `.vhdr` etc can do -- and so you need a dedicated software that allows deeper integration rather than just compatibility. If you want to keep it like this, I think the sentence should be moved up with the rest of the stuff where we discuss what BIDS does.